### PR TITLE
Updating README.md for clarity

### PR DESCRIPTION
**Pull request by nickuncaged1201**
Update a few places in README.md, especially commands regarding how to run tests.

### CODE REVIEW

**Comment by nickuncaged1201** [comment]
r: @pabloem 
r: @manavgarg

File: syndeo-template/README.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ The template works by relying on Beam transforms that implement the `SchemaTrans
 template's classpath.
 
 The template receives a *pipeline spec*, which is a specification defining the pipeline's source, sink, and intermediate
-transforms. Then, the template will take the pipeline spec, and
+transforms. Then, the template will take the pipeline spec, and TODO(nickllx): fill this part.
```
Comment (manavgarg) [inline syndeo-template/README.md:17]: possible to fix the TODO before we submit this change?

**Review by manavgarg — APPROVED** [review]
Thanks Nick for making these changes. This would be extremely helpful for everyone.
LGTM

File: syndeo-template/README.md (hunk 1 above)
Comment (nickuncaged1201) [reply syndeo-template/README.md:17]: @pabloem hey Pablo would you mind filling this TODO here? I'm not a 100% sure about the entire schema transform workflow

Comment (manavgarg) [reply syndeo-template/README.md:17]: can we submit this PR. maybe remove this for now?

Comment (nickuncaged1201) [reply syndeo-template/README.md]: Done

File: syndeo-template/README.md [hunk 2]
```diff
@@ -73,20 +73,22 @@ Supported `SchemaTransform` and configuration parameters are configured in `Synd
 
 ### Run tests (unit and integration tests)
 
-Note that syndeo depends of the teleport integration testing framework, so make sure to install that locally:
+Note that syndeo depends on the teleport integration testing framework, so make sure to install that locally:
 
-```
-mvn install -DskipTests -f pom.xml -pl .,it/pom.xml
+```shell
+mvn install -DskipTests -f pom.xml -pl .,metadata,it/pom.xml
```
Comment (bvolpato) [inline syndeo-template/README.md:79]: Why not `-pl syndeo-template -am` instead of requiring a separate step to install deps?

Comment (nickuncaged1201) [reply syndeo-template/README.md:79]: Neat! I wasn't aware of this way of building it!

File: syndeo-template/README.md [hunk 3]
```diff
@@ -73,20 +73,28 @@ Supported `SchemaTransform` and configuration parameters are configured in `Synd
...
 ```
 
 To **run unit tests** for the Syndeo template, run the following command. Note that this command knows to skip
 integration tests and only runs unit tests:
 
 ```shell
-mvn clean package test -f pom.xml -pl syndeo-template/pom.xml
+mvn clean package test -f pom.xml -pl syndeo-template/pom.xml -Djib.skip=true
```
[the remark this answers is not in the document]
Comment (nickuncaged1201) [reply syndeo-template/README.md]: Just reorg this part a little bit. Please feel free to leave any more comments!

**Comment by nickuncaged1201** [comment]
r: @ bvolpato
@ pabloem

File: syndeo-template/pom.xml [hunk 4]
```diff
@@ -174,8 +174,8 @@
         <dependency>
             <groupId>org.apache.beam</groupId>
             <artifactId>beam-sdks-java-extensions-sql-expansion-service</artifactId>
-            <!--<version>2.46.0-SNAPSHOT</version>-->
-            <version>2.46.0-20230214.212059-18</version>
+            <version>2.46.0-SNAPSHOT</version>
+            <!-- <version>2.46.0-20230214.212059-18</version> -->
```
Comment (manavgarg) [inline syndeo-template/pom.xml:178]: just delete this line.

**Review by manavgarg — APPROVED** [review]

File: syndeo-template/pom.xml (hunk 4 above)
Comment (nickuncaged1201) [reply syndeo-template/pom.xml]: done

**Comment by manavgarg** [comment]
pinging here again.
@pabloem @bvolpato @nickuncaged1201 can we please have this merged? its been pending for a while now.

**Review by bvolpato — APPROVED** [review]

**Review by bvolpato — APPROVED** [review]

**Comment by bvolpato** [comment]
Hello, we have some tooling that needs to re-run all the time there is a change / merge. Please use rebase instead of merge when possible :-) Thanks!

**Comment by nickuncaged1201** [comment]
just rebased again. Please see if it ready to be merged!

**Pull request merged**